djstripe/models.py

```diff
@@ -780,26 +781,26 @@ def _attach_objects_hook(self, cls, data):
 
 
 @class_doc_inherit
-class Plan(StripePlan):
+class Plan(StripePlan): # noqa
+
     __doc__ = getattr(StripePlan, "__doc__")
 
     # account = ForeignKey("Account", related_name="plans")
 
-    class Meta(object):
+    class Meta(object): # noqa
         ordering = ["amount"]
 
     @classmethod
     def get_or_create(cls, **kwargs):
-        """ Get or create a Plan."""
-
+        """Get or create a Plan."""
         try:
             return Plan.objects.get(stripe_id=kwargs['stripe_id']), False
         except Plan.DoesNotExist:
             return cls.create(**kwargs), True
 
     @classmethod
     def create(cls, **kwargs):
-        # A few minor things are changed in the api-version of the create call
+        """Change a few minor things for api create call."""
```

Comment:
"""
Create a plan both locally and on Stripe.

:param kwargs: Keyword arguments for creating a dj-stripe Plan class. These kwargs are then sanitized and passed to the stripe API to create a plan on Stripe.
"""